index.css

```diff
@@ -34,6 +35,29 @@ button {
   gap: 2rem;
   padding: 2rem 4rem;
 }
+.form-image-section {
+  width: calc(50% - 1px);
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 2rem;
+  padding: 2rem 4rem;
+}
+.form-top-image {
+  display: none;
+}
+@media screen and (max-width: 640px) {
```

Comment:
These CSS changes effectively add responsive design elements, improving the user experience on mobile devices.

- Good use of media queries to handle responsive design.
- Consider how descriptive class names can make the purpose of CSS classes clearer.